src/Sulu/Bundle/ContentBundle/Tests/Functional/Controller/NodeControllerTest.php

```diff
@@ -1496,31 +1515,49 @@ public function testCopy()
 
         $client->request(
             'POST',
-            '/api/nodes/' . $data[0]['id'] . '?webspace=sulu_io&language=en&action=copy&destination=' . $data[1]['id']
+            '/api/nodes/' . $data[4]['id'] . '?webspace=sulu_io&language=en&action=copy&destination=' . $data[2]['id']
         );
         $this->assertHttpStatusCode(200, $client->getResponse());
         $response = json_decode($client->getResponse()->getContent(), true);
 
         // check some properties
-        $this->assertNotEquals($data[0]['id'], $response['id']);
-        $this->assertEquals('test1', $response['title']);
-        $this->assertEquals('/test2/test1', $response['path']);
-        $this->assertEquals('/test2/test1', $response['url']);
+        $this->assertNotEquals($data[4]['id'], $response['id']);
+        $this->assertEquals('test5', $response['title']);
+        $this->assertEquals('/test2/test3/test5', $response['path']);
+        $this->assertEquals('/test2/test3/testing5', $response['url']);
 
         // check old node
         $client->request(
             'GET',
-            '/api/nodes/' . $data[0]['id'] . '?webspace=sulu_io&language=en'
+            '/api/nodes/' . $data[4]['id'] . '?webspace=sulu_io&language=en'
         );
         $this->assertHttpStatusCode(200, $client->getResponse());
         $response = json_decode($client->getResponse()->getContent(), true);
 
-        $this->assertEquals($data[0]['id'], $response['id']);
-        $this->assertEquals($data[0]['title'], $response['title']);
-        $this->assertEquals($data[0]['path'], $response['path']);
-        $this->assertEquals($data[0]['template'], $response['template']);
-        $this->assertEquals($data[0]['url'], $response['url']);
-        $this->assertEquals($data[0]['article'], $response['article']);
+        $this->assertEquals($data[4]['id'], $response['id']);
+        $this->assertEquals($data[4]['title'], $response['title']);
+        $this->assertEquals($data[4]['path'], $response['path']);
+        $this->assertEquals($data[4]['template'], $response['template']);
+        $this->assertEquals($data[4]['url'], $response['url']);
+        $this->assertEquals($data[4]['article'], $response['article']);
+
+        $englishRouteNode = $this->session->getNode('/cmf/sulu_io/routes/en/test2/test3/testing5');
+        $this->assertNotEquals(
+            $englishRouteNode->getPropertyValue('sulu:content', PropertyType::REFERENCE),
```

Comment:
doesnt return this always a `NodeInterface` that would be always not equals to a string. in my opinion you would have to call here `getProperty('...')->getSting()` to be sure that you get the uuid.